src/Auth/Login.tsx

```diff
@@ -2,21 +2,20 @@ import React, { useState } from "react";
 import { RouteComponentProps, withRouter } from "react-router-dom";
 
 import Input from "src/components/Input";
-import * as routes from "src/common/constants/pageRoutes";
+import { routes } from "src/common/constants/pageRoutes";
 import Button from "src/components/Button";
 import Link from "src/components/Link";
 import { SignInContainer } from "./Auth.styles";
-import { IParams } from "src/common/interface";
+import { useAuth } from "src/context/GlobalContext";
 
-const Login: React.FunctionComponent<RouteComponentProps<IParams>> = ({
-  history,
-}) => {
+const Login: React.FunctionComponent<RouteComponentProps> = () => {
   const [email, setEmail] = useState<string>("");
   const [password, setPassword] = useState<string>("");
+  const { actions } = useAuth();
 
   const handleSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
     e.preventDefault();
-    history.push(routes.HOME);
+    actions?.signIn({ email, password });
```

Comment:
`signIn` is a required dependency

Reply:
https://github.com/pesto-students/juntos-frontend/pull/23#discussion_r652915079